mesmerize_napari/napari1d_manager.py

```diff
@@ -111,8 +106,17 @@ def callback(layer, event):
 
     def update_visible_components(self):
         edge_colors, face_colors = self.get_colors()
-        self.spatial_layer.edge_color = edge_colors
-        self.temporal_layer.color = edge_colors
+        og_num, current_num = np.shape(edge_colors)[0], np.shape(self.spatial_layer.edge_color)[0]
```

Comment:
more informative names? what is `og_num` and `current_num` referencing?

Reply:
fixed